website/src/components/SchemaFormatTabs.astro

```diff
@@ -131,6 +151,241 @@ function extractCode(props: CodeBlockConfig): string {
+          if (result !== null) {
+            allOfResults.push(result);
+          }
+        }
+      } else {
+        // Process the item directly
+        const result = generateJsonFromSchemaObject(item, visitedRefs, rootSchema);
+        if (result !== null) {
+          allOfResults.push(result);
+        }
+      }
+    }
+    
+    // Merge all the results into a single object
+    if (allOfResults.length > 0) {
+      const mergedResult = allOfResults.reduce((acc: Record<string, unknown>, result) => {
+        if (typeof result === 'object' && result !== null) {
+          return Object.assign(acc, result as Record<string, unknown>);
+        }
+        return acc;
+      }, {});
+      
+      // Also process the current schema's own properties
+      const currentResult = generateJsonFromSchemaObject({ ...schema, allOf: undefined }, visitedRefs, rootSchema);
+      if (currentResult && typeof currentResult === 'object') {
+        return Object.assign(mergedResult, currentResult as Record<string, unknown>);
+      }
+      
+      return mergedResult;
+    }
+    
+    // Fallback: process current schema without allOf
+    return generateJsonFromSchemaObject({ ...schema, allOf: undefined }, visitedRefs, rootSchema);
+  }
+
+  // Handle different schema types
+  switch (schema.type) {
+    case 'object': {
+      const obj: Record<string, unknown> = {};
+      
+      // If this object has examples, try to use them as a template
+      if (schema.examples && schema.examples.length > 0) {
+        const example = schema.examples[0];
+        if (typeof example === 'object' && example !== null) {
+          // Use the example as a template, but still generate missing fields
+          for (const [key, value] of Object.entries(example)) {
+            if (schema.properties && schema.properties[key]) {
+              // Use the example value directly - it's already properly formatted
+              obj[key] = value;
+            }
+          }
+        }
+      }
+      
+      if (schema.properties) {
+        for (const [key, propSchema] of Object.entries(schema.properties)) {
+          const isRequired = schema.required && schema.required.includes(key);
+          
+          // If we already have a value from examples, use it
+          if (obj[key] !== undefined) {
+            continue;
+          }
+          
+          const exampleValue = generateJsonFromSchemaObject(propSchema as JsonSchema, visitedRefs, rootSchema);
+          
+          // Include required fields and optional fields that have valid examples
+          if (isRequired || (exampleValue !== null && exampleValue !== undefined)) {
+            obj[key] = exampleValue;
+          }
+        }
+      }
+      return obj;
+    }
+
+    case 'array':
+      if (schema.items) {
+        const itemExample = generateJsonFromSchemaObject(schema.items as JsonSchema, visitedRefs, rootSchema);
+        return [itemExample];
+      }
```

Comment:
As I was reviewing this, I noticed that this file doesn't seem to be adhering to the `prettier` standards that other TypeScript files in the website project are (e.g. single quotes, line wrapping, etc.) and realized it was because we don't have the `prettier-plugin-astro` added as a dev dependency yet.

Can you add this and update `.prettierrc` to the following:

```
{
  "trailingComma": "all",
  "tabWidth": 2,
  "plugins": ["prettier-plugin-astro"],
  "overrides": [
    {
      "files": "*.astro",
      "options": {
        "parser": "astro"
      }
    }
  ]
}
```

Then these formatting discrepancies should get resolved.